name: lovo_robot_bridge

topics:
  # Domain 70 → 60 (PC → Robot1)
  /robot1/target_angles:
    type: std_msgs/msg/Float64MultiArray
    from_domain: 70
    to_domain: 60
    remap: /target_angles

  /robot1/servo_status:
    type: std_msgs/msg/Bool
    from_domain: 70
    to_domain: 60
    remap: /servo_status

  /robot1/gripper_control:
    type: std_msgs/msg/Int32
    from_domain: 70
    to_domain: 60
    remap: /gripper_control

  /robot1/target_coords:
    type: std_msgs/msg/Float64MultiArray
    from_domain: 70
    to_domain: 60
    remap: /target_coords

  # Domain 60 → 70 (Robot1 → PC)
  /current_angles:

    type: std_msgs/msg/Float64MultiArray
    from_domain: 60
    to_domain: 70
    remap: /robot1/current_angles

  /current_coords:
    type: std_msgs/msg/Float64MultiArray
    from_domain: 60
    to_domain: 70
    remap: /robot1/current_coords

  /current_pose:
    type: geometry_msgs/msg/Pose
    from_domain: 60
    to_domain: 70
    remap: /robot1/current_pose

  /robot1/goal_pose:
    type: geometry_msgs/msg/Pose
    from_domain: 70
    to_domain: 60
    remap: /goal_pose

  # Domain 70 → 61 (PC → Robot2)
  /robot2/target_angles:
    type: std_msgs/msg/Float64MultiArray
    from_domain: 70
    to_domain: 61
    remap: /target_angles

  /robot2/servo_status:
    type: std_msgs/msg/Bool
    from_domain: 70
    to_domain: 61
    remap: /servo_status

  /robot2/gripper_control:
    type: std_msgs/msg/Int32
    from_domain: 70
    to_domain: 61
    remap: /gripper_control

  /robot2/target_coords:
    type: std_msgs/msg/Float64MultiArray
    from_domain: 70
    to_domain: 61
    remap: /target_coords

  # Domain 61 → 70 (Robot2 → PC)
  /current_angles:
    type: std_msgs/msg/Float64MultiArray
    from_domain: 61
    to_domain: 70
    remap: /robot2/current_angles

  /current_coords:
    type: std_msgs/msg/Float64MultiArray
    from_domain: 61
    to_domain: 70
    remap: /robot2/current_coords

  /current_pose:
    type: geometry_msgs/msg/Pose
    from_domain: 61
    to_domain: 70
    remap: /robot2/current_pose

  /robot2/goal_pose:
    type: geometry_msgs/msg/Pose
    from_domain: 70
    to_domain: 61
    remap: /goal_pose
